### ISSUE TITLE
Hourly Demand output values are 10-times higher than the accumulated power of all appliances

### ISSUE THREAD

**Issue by a-linke** [rl-institut/RAMP#24]
Hi @Bachibouzouk , I observed that the values for the power demand in the yearly_profile_hourly_resolution.xlsx file produced by running `python ramp.py -i ramp/input_files/chps -n 3 -y 2022
`
are approx 10-times higher than the accumulated power of all listed appliances in the input files 1chps.xlsx - 12chps.xlsx

**Comment by Bachibouzouk** [comment]
Is it *exactly* 12 times?

**Comment by Bachibouzouk** [comment]
I think I found the error, in the conversion form minutes to hours, I mistakenly just summed the values for each minute instead of averaging it.

The file `'yearly_profile_min_resolution.csv'` is the original one, the other one is simply using pandas `resample` method
You can play with it yourself in a jupyter notebook or in a python script
```
import pandas as pd
df = pd.read_csv("yearly_profile_min_resolution.csv", index_col=0)
df.index = pd.to_datetime(df.index, infer_datetime_format=True)
df.resample("H").mean()
```

**Comment by Bachibouzouk** [comment]
Fixed in 94c7f15c691603c966f28fb162bbb2404719c4c5
